### PR TITLE
Ticket #156 - 404 Page

### PR DESCRIPTION
**Pull request by nym1k**
Custom 404,500 error pages
New routes for errors with controller and views

### CODE REVIEW

File: app/controllers/countries_controller.rb [hunk 1]
```diff
@@ -4,6 +4,8 @@ class CountriesController < ApplicationController
   def show
     @country = GeoEntity.find_by(name: country_name_from_param(params[:name]))
 
+    return redirect_to(action: 'unprocessable_entity', controller: 'errors') unless @country
```
Comment (WKocur) [inline app/controllers/countries_controller.rb:7]: I would probably redirect to not_found here. What do you think?

Comment (nym1k) [reply app/controllers/countries_controller.rb]: Agreed (think I missed this after testing)

**Comment by WKocur** [comment]
I think you need this `config.exceptions_app = self.routes` in `application.rb`. You can test what it will do on prod by setting `config.consider_all_requests_local` to false in `environments/development.rb`

**Comment by WKocur** [comment]
Have you tested an incorrect url that doesn't go to the country controller? That is what didn't work for me.

**Pull request merged**